src/Sarif/Core/WebMessageUtilities.cs

```diff
@@ -31,18 +31,19 @@
         // "target" field on the request line is a valid URI. Nor do we verify that the request body or response
         // body contains only "VCHAR"s (visible characters as opposed to control characters), as required by
         // the RFC.
-        internal const string CRLF = "\r\n";
+        internal static string NewLineForWindows = "\r\n";
```

Comment:
>NewLineForWindows [](http://example.com/codeflow?start=31&length=17)

Based on Environment.Newline, let's call this WindowsNewline. NonWindowsNewline.

Non-Windows is the actual term.

Reply:
fixed